src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class MovieRequestDto {
+    @NotNull
+    @Min(value = 4)
```

Comment:
Can you use `Min` here? 
![image](https://user-images.githubusercontent.com/32339836/236235683-056594d2-ec34-4f7c-867b-dad0a9b8e850.png)